lib/linguist/languages.yml

```diff
@@ -4995,6 +4994,15 @@ Ruby:
   - Vagrantfile
   - buildfile
   language_id: 326
+Ruby Gemfile.lock:
+  type: data
+  color: "#701516"
+  searchable: false
```

Comment:
@lildude: What does `searchable` do? It's on e.g. JSON but github.com/search?q=language:JSON&type=code works.

Reply:
https://github.com/github/linguist/issues/5269